README.md

```diff
@@ -8,7 +8,42 @@ components.
 
 You will need at least the following installed:
 
- * LLVM 15.
- * wasm-ld 15 (this is part of LLVM, but your distribution might split it into
-   a separate package: your wasm-ld version must match your LLVM version).
- * typescript 5.2 or later.
+ * LLVM 15. Depending on your distribution you may need to install `llvm-devel`
+   or `llvm-extra` to get necessary extra parts such as `wasm-ld`.
+ * typescript (`tsc`) 5.2 or later. If your distribution doesn't package
+   typescript, you can install your own with `npm install
+   --prefix=/path/to/install typescript`.
+ * `yarn`
```

Comment:
When I set up a new dev environment, I look at the Dockerfile and replay those commands, modifying as necessary for systems that don't use apt. It might be helpful to point people to that approach. Also, the Dockerfile does significantly more, with gradle, java, playwright, sdkman, etc. I think all of those are needed to get a working development environment for making and testing code changes across the system, but maybe these instructions are aiming at something else?